packages/ui/components/Footer.tsx

```diff
@@ -1,5 +1,41 @@
+import { ReactNode } from 'react'
+
+import { ArrowUpRight, Code, Discord, Twitter } from '@dao-dao/icons'
+
 export interface FooterProps {}
 
+const FooterItem = ({ icon, name }: { icon: ReactNode; name: string }) => (
+  <div className="flex flex-row justify-between items-center p-2 hover:bg-background-interactive-hover rounded-md cursor-pointer">
+    <div className="flex flex-row gap-2 items-center text-text-tertiary">
+      <div>{icon}</div>
+      <div>{name}</div>
+    </div>
+
+    <ArrowUpRight className="text-text-tertiary" height={12} width={12} />
+  </div>
+)
+
 export const Footer = ({}: FooterProps) => {
-  return <p>placeholder</p>
+  return (
+    <div className="max-w-sm">
+      <div className="py-4 text-text-tertiary">
+        Website is currrently in beta. Please let the developers know if you
+        encounter any issues.
+      </div>
+      <div className="flex flex-col">
+        <FooterItem
+          icon={<Code className="text-text-tertiary" />}
+          name="Documentation"
+        />
+        <FooterItem
+          icon={<Twitter className="text-text-tertiary" />}
+          name="Twitter"
+        />
+        <FooterItem
+          icon={<Discord className="text-text-tertiary" />}
+          name="Discord"
+        />
+      </div>
```

Comment:
```suggestion
      <div className="flex flex-col">
        <FooterItem
          icon={<Code className="w-4 h-4" />}
          name="Documentation"
        />
        <FooterItem
          icon={<Twitter className="w-4 h-4" />}
          name="Twitter"
        />
        <FooterItem
          icon={<Discord className="w-4 h-4" />}
          name="Discord"
        />
      </div>
```

`FooterItem` already seems to wrap the icon and name in a `div` with `text-text-tertiary`, so we can let all the icons inherit the color by default. Also, it's good practice to set explicit sizes for icons since they may be sized differently internally, and a consistent size ensures the text is lined up nicely.